src/Assets/LightSkyDefense/Scripts/Enemies/Enemy.cs

```diff
@@ -103,6 +103,8 @@ public void OnDie()
             explodeEffectInstance.gameObject,
             explodeEffectInstance.main.duration + explodeEffectInstance.main.startLifetime.constantMax
         );
+
+        GameObject.Find("Scoreboard").GetComponent<Scoreboard>().PointGain(PointValue);
```

Comment:
We should use messages instead of hardcoded dependencies to track the player score